python_modules/libraries/dagstermill/dagstermill/manager.py

```diff
@@ -175,6 +176,12 @@ def reconstitute_job_context(
             # Set this flag even though we're not in test for clearer error reporting
             raise_on_error=True,
         ) as job_context:
+            known_state = None
+            if dagster_run.parent_run_id:
+                known_state = KnownExecutionState.build_for_reexecution(
+                    instance=instance,
+                    parent_run=instance.get_run_by_id(dagster_run.parent_run_id)
```

Comment:
`instance.get_run_by_id(dagster_run.parent_run_id)` needs to become `check.not_none(instance.get_run_by_id(dagster_run.parent_run_id))`

Reply:
Thanks @smackesey, I'll fix it soon.

Reply:
I actually have made the changes locally and can push to your branch if given permission (these last linting steps can be a real hassle to go back and forth on)-- would you consider doing this: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests